hack/verify-all.sh

```diff
@@ -98,6 +98,12 @@ if [[ "${VERIFY_STAGES:-true}" == "true" ]]; then
   "${ROOT_DIR}"/hack/verify-stages.sh || failed+=(stages)
 fi
 
+if [[ "${VERIFY_DRY_RUN_TESTDATA:-true}" == "true" ]]; then
+  echo "[*] Verifying testdata..."
+  chmod +x "${ROOT_DIR}"/hack/verify-testdata.sh
```

Comment:
This doesn't need to be added here, you need to run it locally, and then `git add hack/verify-testdata.sh`